apps/staking/app/register/[nodeId]/shared/RegistrationNotice.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRegistrationWizard } from '@/app/register/[nodeId]/Registration';
+import { Notice } from '@/components/Notice';
+import { WizardSectionDescription } from '@/components/Wizard';
+import { PREFERENCE } from '@/lib/constants';
+import { ButtonDataTestId, CheckboxDataTestId } from '@/testing/data-test-ids';
+import { useTranslations } from 'next-intl';
+
+export function RegistrationNotice() {
+  const dict = useTranslations('infoNotice');
+  const { setAcceptedNotice } = useRegistrationWizard();
+
+  return (
+    <Notice
+      onContinue={() => setAcceptedNotice(true)}
+      dontShowAgainPreference={PREFERENCE.INFO_NOTICE_DONT_SHOW_REGISTER}
+      confirmButtonDataTestId={ButtonDataTestId.Registration_Notice_Continue}
+      dontShowAgainDataTestId={CheckboxDataTestId.Notice_Registration_Dont_Show_Again}
+    >
+      <WizardSectionDescription
+        description={dict.rich('register', { linkOut: '' })}
```

Comment:
Why the empty `linkOut` here?

Reply:
its an annoying limitation, we need to explicitly say to render nothing in the linkout, then WizardSectionDescription straps the linkOut to the end